pom.xml

```diff
@@ -404,6 +404,11 @@
 				<artifactId>archaius-core</artifactId>
 				<version>${archaius.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>${gs-test.groupId}</groupId>
+				<artifactId>gs-test</artifactId>
+				<version>0.1.12</version>
```

Comment:
Please note that this version of `gs-test` must be built before this MR can be merged.

Reply:
`gs-test:0.1.12` is now built.